packages/internal/src/build/babel/common.ts

```diff
@@ -0,0 +1,14 @@
+import type { TransformOptions } from '@babel/core'
+
+export interface RequireHookOptions extends TransformOptions {
+  extensions?: string[]
+  cache?: boolean
+}
+
+// @NOTE
```

Comment:
I tend to use  `// NOTE:` or `// TODO:` but I've never really thought about the variation of these.

Reply:
I think they're all valid I believe, conventions vary a little, but Github seems to be happy with it :)

Reply:
I guess my question is should we consider a standard for these? I hate those kinds of rules - and I mostly see it as conversational / some people search for `// TODO` or `// @TODO` and then say things like "omg, we have 4394 todos!"